mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -303,6 +312,10 @@ class KMeans @Since("1.5.0") (
   @Since("1.5.0")
   def setSeed(value: Long): this.type = set(seed, value)
 
+  /** @group setParam */
+  @Since("2.1.0")
+  def setInitialModel(value: KMeansModel): this.type = set(initialModel, value)
```

Comment:
It looks like the `fromCenters` method never got implemented. I think this is important since otherwise users can only set an initial model if they already have a trained KMeans model.